README.md

```diff
@@ -23,17 +23,27 @@ In case you missed it, [download the 2019 Phoniebox Calendar PDF here](https://d
 
 * **Gitter Community** we got ourselves a gitter community; chat us up at https://gitter.im/phoniebox
 
-* **Phoniebox 2.0 rc8 released (2020-01-18 ... getting closer to the actual release...)
+* **Phoniebox [2.0](https://github.com/MiczFlor/RPi-Jukebox-RFID/milestone/1?closed=1) released (2020-05-xx)**
 
```

Comment:
I would like to add a line here:
The 2.0 release was pushed through the doors with *countless additions and great oversight* from @s-martin. Thank you. And *substantial input* by @ZyanKLee and [many more contributors](https://github.com/MiczFlor/RPi-Jukebox-RFID/pulls?q=is%3Apr) like, in alphabetical particular order: @andreasbrett @BerniPi  @danielhammer @drocx @fredg02 @Groovylein @lovethisshit @MalteHST @Piehti @SimonChelkowski @veloxidSchweiz and starring @splitti as *the great hotfix*.